includes/gutenberg/shortcodes/shortcodes-to-blocks.php

```diff
@@ -0,0 +1,421 @@
+		if ( $form['attached_page'] != 'none' ) {
+			$forms[ $form_slug ] = $form['name'];
+		}
+	}
+	wp_localize_script( 'bf-embed-form', 'buddyforms_post_forms', $forms );
+
+	wp_localize_script( 'bf-embed-form', 'buddyforms_create_new_form_url', get_admin_url( get_current_blog_id(), 'post-new.php?post_type=buddyforms' ) );
+
+	// All WordPress User Roles
+	$roles = buddyform_get_role_names();
+	wp_localize_script( 'bf-embed-form', 'buddyforms_roles', $roles );
+
+
+	//
+	// Embed a form
+	//
+	register_block_type( 'buddyforms/bf-embed-form', array(
+		'attributes'      => array(
+			'bf_form_slug' => array(
+				'type' => 'string',
+			)
+		),
+		'editor_script'   => 'bf-embed-form',
+		'render_callback' => 'buddyforms_block_render_form',
+	) );
+
+	//
+	// Embed a navigation
+	//
+	register_block_type( 'buddyforms/bf-navigation', array(
+		'attributes'      => array(
+			'bf_form_slug'     => array(
+				'type' => 'string',
+			),
+			'bf_nav_style'     => array(
+				'type' => 'string',
+			),
+			'bf_label_add'     => array(
+				'type'    => 'string',
+				'default' => 'Create'
+			),
+			'bf_label_view'    => array(
+				'type'    => 'string',
+				'default' => 'Display'
+			),
+			'bf_nav_separator' => array(
+				'type'    => 'string',
+				'default' => ' | '
+			),
+
+		),
+		'editor_script'   => 'bf-embed-form',
+		'render_callback' => 'buddyforms_block_navigation',
+	) );
+
+	//
+	// Embed a login form
+	//
+	register_block_type( 'buddyforms/bf-embed-login-form', array(
+		'attributes'      => array(
+			'bf_form_slug'    => array(
+				'type' => 'string',
+			),
+			'bf_redirect_url' => array(
+				'type' => 'string',
+			),
+			'bf_title'        => array(
+				'type' => 'string',
+			),
+
+		),
+		'editor_script'   => 'bf-embed-form',
+		'render_callback' => 'buddyforms_block_render_login_form',
+	) );
+
+	//
+	// Embed a password reset form
+	//
+	register_block_type( 'buddyforms/bf-password-reset-form', array(
+		'attributes'      => array(
+			'bf_redirect_url' => array(
+				'type' => 'string',
+			),
+		),
+		'editor_script'   => 'bf-embed-form',
+		'render_callback' => 'buddyforms_block_password_reset_form',
+	) );
+
+	//
+	// Embed a post list
+	//
+	register_block_type( 'buddyforms/bf-list-submissions', array(
+		'attributes'      => array(
+			'bf_form_slug'        => array(
+				'type' => 'string',
+			),
+			'bf_rights'           => array(
+				'type'    => 'string',
+				'default' => 'public'
+			),
+			'bf_by_author'        => array(
+				'type' => 'string',
+			),
+			'bf_author_ids'       => array(
+				'type' => 'string',
+			),
+			'bf_by_form'          => array(
+				'type' => 'string',
+			),
+			'bf_posts_per_page'   => array(
+				'type' => 'string',
+			),
+			'bf_list_posts_style' => array(
+				'type' => 'string',
+			),
+		),
+		'editor_script'   => 'bf-embed-form',
+		'render_callback' => 'buddyforms_block_list_submissions',
+	) );
+}
+
+add_action( 'init', 'buddyforms_shortcodes_to_block_init' );
+
+//
```

Comment:
This comment block is incorrect to use in php files to describe a function. 

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods 
The above is a guideline to get an idea.